packages/backend-common/src/s3.ts

```diff
@@ -179,16 +178,14 @@ export const getObjectWithPresignedUrl = async (
 
 export const uploadObjectWithPresignedUrl = async (
 	presignedUrl: string,
-	metadata: MediaMetadata,
+	path: string,
 ): Promise<void> => {
-	logger.info(
-		`Uploading with ${presignedUrl}, metadata ${JSON.stringify(metadata)}`,
-	);
-	const fileStream = createReadStream(metadata.mediaPath);
+	logger.info(`Uploading with ${presignedUrl}, file ${path}`);
```

Comment:
whilst these are our logs, the pre-signed url contains a secret (and is also very long), so wonder if its worth redacting it a bit? perhaps putting the path first in the message too, if the filename is valuable?

Reply:
ah agreed it shouldn't be logged. It was helpful logging it when trying to debug why the uploads were failing - but that can all be done locally.